### ISSUE TITLE
UserWarning: Implicit dimension choice for softmax has been deprecated. [SSD Forward Pass]

### ISSUE THREAD

**Issue by santhoshdc1590** [amdegroot/ssd.pytorch#104]
User Warning on running SSD Forward Pass

`ssd.pytorch/ssd.py:100: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  self.softmax(conf.view(-1, self.num_classes)),  # conf preds`

This error message can be seen on the [Source code for torch.nn.functional](http://pytorch.org/docs/master/_modules/torch/nn/functional.html)

**Comment by amdegroot** [comment]
Just added this fix by passing dim in nn.Softmax, thanks for pointing it out.
